languages/csharp/AppEncryption/README.md

```diff
@@ -1,192 +1,66 @@
-    /// <returns>The persistence key associated with the stored Data Row Record</returns>
-    public virtual string Store(TP payload, Persistence<TD> dataPersistence)
-    {
-        TD dataRowRecord = Encrypt(payload);
-        return dataPersistence.Store(dataRowRecord);
-    }
-
-    /// <summary>
-    /// Encrypts a payload, stores the resulting Data Row Record into the provided data persistence store with
-    /// given key
-    /// </summary>
-    /// <param name="key">Key against which the encrypted DRR will be saved</param>
-    /// <param name="payload">Payload to be encrypted</param>
-    /// <param name="dataPersistence">The persistence store where the encrypted DRR should be stored</param>
-    public virtual void Store(string key, TP payload, Persistence<TD> dataPersistence)
-    {
-        TD dataRowRecord = Encrypt(payload);
-        dataPersistence.Store(key, dataRowRecord);
-    }
-```
-
-Example `Dictionary`-backed `Persistence` implementation:
-
-```c#
-public class DictionaryPersistence : Persistence<JObject>
-{
-    Dictionary<string, JObject> dictionaryPersistence = new Dictionary<string, JObject>();
-
-    public override Option<JObject> Load(String key)
-    {
-        return dictionaryPersistence.TryGetValue(key, out JObject result) ? result : Option<JObject>.None;
-    }
-
-    public override void Store(String key, JObject value)
-    {
-        dictionaryPersistence.Add(key, value);
-    }
-}
-```
-
-Putting it all together, an example end-to-end use of the store and load calls:
-
-```c#
-// Encrypts the payload, stores it in the dictionaryPersistence and returns a look up key
-string persistenceKey = appEncryptionJsonImpl.Store(originalPayload.ToJObject(), dictionaryPersistence);
-
-// Uses the persistenceKey to look-up the payload in the dictionaryPersistence, decrypts the payload if any and then returns it
-Option<JObject> payload = appEncryptionJsonImpl.Load(persistenceKey, dictionaryPersistence);
-```
-
-#### Plain Encrypt/Decrypt Style
-This usage style is similar to common encryption utilities where payloads are simply encrypted and decrypted, and it is completely up to the calling application for storage responsibility.
-
-```c#
-string originalPayloadString = "mysupersecretpayload";
-
-// encrypt the payload
-byte[] dataRowRecordBytes = appEncryptionBytes.Encrypt(Encoding.UTF8.GetBytes(originalPayloadString));
-
-// decrypt the payload
-string decryptedPayloadString = Encoding.UTF8.GetString(appEncryptionBytes.Decrypt(newDataRowRecordBytes));
-```
-
-
-### Metastore
-Please refer to the [Java lib's notes](../../java/app-encryption#metastore) until documentation cleanup/refactor is complete. It contains deployment info, data size estimates, etc. Until then, we will simply provide code usage examples here.
+Detailed information about the Metastore, including any provisioning steps, can be found [here](../../../docs/Metastore.md).
 
 #### ADO Metastore
```

Comment:
let's make this `RDBMS` and we mention below it happens to use ADO for the implementation